packages/nimble-components/src/table/components/group-row/index.ts

```diff
@@ -29,6 +27,9 @@ export class TableGroupRow extends FoundationElement {
     @observable
     public nestingLevel = 0;
 
+    @observable
+    public rowStateIndex?: number;
```

Comment:
Renamed to `resolvedRowIndex` and added a comment. (Thought `visualRowIndex` could end up being confusing since the visual / actually rendered rows in the table are a subset due to the virtualization.)